menu.html

```diff
@@ -4,57 +4,115 @@
+        <link rel="stylesheet" href="https://use.typekit.net/kay2rdn.css" />
+        <link
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+            rel="stylesheet"
+        />
+
         <title>Restaurant</title>
-        <link rel="stylesheet" href="src/navbar.css">
-        <link rel="stylesheet" href="src/menu.css">
     </head>
 
     <body>
         <nav>
-            <h1>RESTAURANT</h1>
+            <h1>American Steakhouse</h1>
             <a href="index.html">Home</a>
             <a href="menu.html" class="active">Menu</a>
-            <a href="about.html">About</a>
+            <div>
+                <button>About</button>
+                <div class="dropdown">
+                    <a href="#">Kitchen</a>
+                    <a href="#">Chefs</a>
+                    <a href="#">Wijngozers</a>
+                    <a href="#">Location</a>
+                </div>
+            </div>
             <a href="contact.html">Contact</a>
         </nav>
 
-        <table>
-            <tr>
-                <td><h3>Menu item 1</h3>test</td>
-                <td>&euro; 0.99</td>
-            </tr>
-            <tr>
-                <td><h3>Menu item 2</h3>test</td>
-                <td>&euro; 2.99</td>
-            </tr>
-            <tr>
-                <td><h3>Menu item 3</h3>test</td>
-                <td>&euro; 2.99</td>
-            </tr>
-            <tr>
-                <td><h3>Menu item 4</h3>test</td>
-                <td>&euro; 2.99</td>
-            </tr>
-            <tr>
-                <td><h3>Menu item 5</h3>test</td>
-                <td>&euro; 2.99</td>
-            </tr>
-            <tr>
-                <td><h3>Menu item 6</h3>test</td>
-                <td>&euro; 2.99</td>
-            </tr>
-            <tr>
-                <td><h3>Menu item 7</h3>test</td>
-                <td>&euro; 2.99</td>
-            </tr>
-            <tr>
-                <td><h3>Menu item 8</h3>test</td>
-                <td>&euro; 2.99</td>
-            </tr>
-        </table>
-
+        <main>
+            <h2>Steaks</h2>
+            <table>
+                <tr>
+                    <td><h3>Rib eye</h3>Beatiful rib eye soaked in a barbecue inspired marinade.</td>
+                    <td>&euro; 24.-</td>
+                </tr>
+                <tr>
+                    <td><h3>Garlic-butter steak</h3>A delicious steak cooked to your liking from the top sirloin.</td>
+                    <td>&euro; 22.-</td>
+                </tr>
+                <tr>
+                    <td><h3>New York strip steak</h3>Tender piece of meat from the top part of the short loin, cooked New York style.</td>
+                    <td>&euro; 27.-</td>
+                </tr>
+                <tr>
+                    <td><h3>Filet mignon</h3>Delicious piece of meat from the tip of the tenderloin.</td>
+                    <td>&euro; 29.-</td>
+                </tr>
+            </table>
+            <h2>Ribs</h2>
```

Comment:
We kunnen hier ook een caption van maken, zodat het beter is voor de table leesbaarheid
```suggestion
            <caption>Ribs</caption>
```

Reply:
gedaan